controllers/articles.js

```diff
@@ -1,10 +1,11 @@
 import crypto from 'crypto';
 import models from '../models/index';
+import readTime from '../helpers/readingTime';
 
 const {
   article, User, bookmark, likes
 } = models;
-const attributes = ['title', 'body', 'description', 'slug', 'createdAt', 'updatedAt', 'ratings', 'categories', 'tagList'];
+const attributes = ['title', 'body', 'description', 'slug', 'createdAt', 'updatedAt', 'ratings', 'categories', 'readtime', 'tagList'];
```

Comment:
Respecting to Airbnb naming convention, it's better to name your object in camelCase